### DISCUSSION TITLE
IS THIS ENGINE IS A VIRUS OR NORTON'S IQ = 0?

### DISCUSSION THREAD

**Discussion by abjdev** [KadeArchive/Kade-Engine#2373] [General]
![image](https://user-images.githubusercontent.com/91409956/134800084-b23a2b89-79b8-4408-8f1b-a45465b659a5.png)

**Comment by EcoVeni** [comment]
kade engine is not a virus

**Comment by Jotaro-Gaming** [comment]
yeah kade aint a virus this is happening because kade dev isnt a trusted devolper by microsoft

**Comment by isakube** [comment]
norton 0 iq

**Reply by abjdev** [reply]
ofc

**Comment by abjdev** [comment]
Yes it is not a virus. I reinstalled the Kade Engine and it works.

**Comment by ACrazyTown** [comment]
It's not a virus, but it's probably starting to be recognized as one because people are making FNF mods shutdown PCs and do stuff like that
